manifest/net.NepuShiro/ChangeJoinLeaveSound/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "ChangeJoinLeaveSound",
+	"id": "net.NepuShiro.ChangeJoinLeaveSound",
+	"description": "Allows you to change the default UserJoin/Leave sounds, and color.",
+	"category": "Dashboard",
+	"sourceLocation": "https://github.com/NepuShiro/ChangeJoinLeaveSound",
+	"versions": {
+		"1.0.0": {
```

Comment:
Non-blocking: built in debug mode

Reply:
Additional assembly attributes were also not set `[assembly: AssemblyDescription("Description of your mod")]` and
`[assembly: AssemblyCopyright("Copyright © 2024 ExampleAuthor")]`

Reply:
This might go for most of the assemblies, I can rebuild them with those filled if need be

Reply:
Looks like it is the same for several of the others as well.